packages/web/src/components/cells/code.tsx

```diff
@@ -257,7 +259,7 @@ export default function CodeCell(props: {
           ) : (
             <ResizablePanelGroup direction="vertical">
               <ResizablePanel style={{ overflow: 'scroll' }} defaultSize={60}>
-                <div className={cn(cellMode !== 'off' && 'opacity-50')}>
+                <div className={cn(cellMode !== 'off' && 'opacity-50')} id={cell.filename}>
```

Comment:
Are there characters supported in filenames that would not be supported by html ids (at least without escaping)?

Can we use the cell id instead? Right now it changes between server restarts but we definitely need to fix that.

Reply:
The server and client are communicating over a session, so they both always know what the current session is.